src/main/scala/org/clulab/wm/eidos/utils/MentionUtils.scala

```diff
@@ -0,0 +1,20 @@
+package org.clulab.wm.eidos.utils
+
+import org.clulab.odin._
+
+object MentionUtils {
+
+  def newCrossSentenceMention(m: CrossSentenceMention, attachments: Set[Attachment]) =
+      new CrossSentenceMention(m.labels, m.anchor, m.neighbor, m.arguments, m.document, m.keep, m.foundBy, attachments)
+
+  def withAttachments(mention: Mention, seqOfAttachments: Seq[Attachment]): Mention = {
+    val setOfAttachments = seqOfAttachments.toSet
+
+    mention match {
+      case m: TextBoundMention => m.copy(attachments = setOfAttachments)
```

Comment:
@kwalcock  I think we might need this to be `m.copy(attachments = m.attachments ++ setOfAttachments)` or something similar (or added up in line 11).  I think as it it would overwrite any attachments that were previously there.  I don't think it is "critical" here because hedging is only done on event mentions and they don't usually (so far ever?) have attachments before hedging, but if we want it to be more general purpose then I think the change needs to be made...? There's a chance I'm wrong but I think this is the way this would work

Reply:
and for the other lines too...

Reply:
Uh oh.  I think you're right and I remember thinking about the ++ but don't see it anywhere.  I'll change and test.  Thanks for the keen powers of observation.